transport/src/main/java/io/scalecube/transport/TransportConfig.java

```diff
@@ -82,7 +91,20 @@ public Builder useNetworkEmulator(boolean useNetworkEmulator) {
       return this;
     }
 
+    public Builder withMessageCodec(MessageCodec messageCodec) {
+      this.messageCodec = messageCodec;
+      return this;
+    }
+
+    /**
+     * Finish configuration.
+     *
+     * @return transport config
+     */
     public TransportConfig build() {
+      if (messageCodec == null) {
```

Comment:
i suggest to put it on field as default so here will be nothing...
so unless specified to another message codec jackson will be used.

Reply:
sure it will be better